common/pb/session_commands.proto

```diff
@@ -160,3 +163,30 @@ message Command_AccountPassword {
     optional string old_password = 1;
     optional string new_password = 2;
 }
+
+message Command_ForgotPasswordRequest {
+    extend SessionCommand {
+        optional Command_ForgotPasswordRequest ext = 1021;
+    }
+    optional string user_name = 1;
+    optional string clientid = 2;
+}
+
+message Command_ForgotPasswordReset {
+    extend SessionCommand {
+        optional Command_ForgotPasswordReset ext = 1022;
+    }
+    optional string user_name = 1;
```

Comment:
Why are these all optional? At least user_name makes sense to be required, probably token also and new_password